tests/dbfs/test_api.py

```diff
@@ -103,6 +110,27 @@ def test_list_files_does_not_exist(self, dbfs_api):
 
         assert len(files) == 0
 
+    def test_list_files_rate_limited(self, dbfs_api):
+        rate_limit_exception = get_rate_limit_exception()
+        # Simulate 2 rate limit exceptions followed by a full successful list operation
+        exception_sequence = [rate_limit_exception, rate_limit_exception, None]
+        dbfs_api.client.list_files = mock.Mock(side_effect=exception_sequence)
+        # Should succeed
+        files = dbfs_api.list_files(TEST_DBFS_PATH)
+
+        assert len(files) == 0
+
+    def test_mkdirs_rate_limited(self, dbfs_api):
+        rate_limit_exception = get_rate_limit_exception()
+        # Simulate 2 rate limit exceptions followed by a full successful list operation
```

Comment:
"successful ~~list~~ **mkdir** operation"

Maybe it's easier to just not mention the operation type in the comment.